fdbserver/KeyValueStoreMemory.actor.cpp

```diff
@@ -464,7 +464,7 @@ class KeyValueStoreMemory final : public IKeyValueStore, NonCopyable {
 
 	// Data format for normal operation:
 	// +-------------+-------------+-------------+--------+--------+
-	// | opType      | len1        | len2        | param2 | param2 |
+	// | opType      | len1        | len2        | param1 | param2 |
```

Comment:
nit: I changed the format in my pr, this would cause conflicts so maybe revert for now?

Reply:
Lets see who gets the fun of rebasing :D